From d20abf491a0ad9db1616012b19afe023b4607975 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 16 Jan 2023 13:59:14 +0100
Subject: [PATCH 0110/1141] arm64: dts: st: enable RTC on stm32mp257f-ev board

Enable RTC on stm32mp257f-ev board.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I68942fb34d8812ceafdc431a3b4f91c479580a12
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/303722
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
index 44a8d4eb4a25..e0a4cfa1fa5f 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
@@ -37,6 +37,10 @@ &arm_wdt {
 	status = "okay";
 };
 
+&rtc {
+	status = "okay";
+};
+
 &scmi_regu {
 	scmi_v3v3: voltd-v3v3 {
 		reg = <VOLTD_SCMI_STPMIC2_BUCK7>;
-- 
2.39.2

